### PR TITLE
Integrate 12/2/22

### LINKED ISSUE #10072: Defork RectangleEdges.h
### Problem Description

_Creating issue to track deforking of this file_

Recent integration came across this PR (https://github.com/facebook/react-native/commit/56e9aa369f5c13af38cf80ba47e9eb29d835ec89) added these lines of code to [RectangleEdges.h](https://github.com/facebook/react-native/commit/56e9aa369f5c13af38cf80ba47e9eb29d835ec89#diff-5a2469bf2c3c7e2c071dfe360aa9d9b21e664782f73de942858a353c125680ee)

```
struct RectangleEdges {
 ...
 static RectangleEdges<T> const ZERO;
};

template <typename T>
constexpr RectangleEdges<T> const RectangleEdges<T>::ZERO = {};
```

which created these C2370 redefinition; different stage class errors 

image

Removing the constexpr identifier solves the problem but not sure that's the solution. I think it has to do with how we load in header files into our project since this did not break upstream.

### Steps To Reproduce

1. Delete forked RectangleEdges.h file in RNW repo
2. Try to build the playground solution

### Expected Results

Should build fine

### CLI version

npx react-native --version

### Environment

```markdown
npx react-native info
```

### Target Platform Version

10.0.19041

### Target Device(s)

Desktop

### Visual Studio Version

Visual Studio 2019

### Build Configuration

Debug

### Snack, code example, screenshot, or link to a repository

_No response_

### PR DESCRIPTION
**Pull request by TatianaKapos**
## Description
https://github.com/facebook/react-native/compare/81e441ae8...dfbb63996

### Type of Change
Integration

### What

- [rename Geometry.h -> Vector.h](https://github.com/facebook/react-native/commit/e6cba997443ad72961cee1eee39346f80b48184e)
- [chore: renamed react-native-codegen to @react-native/codegen](https://github.com/facebook/react-native/commit/b7a85b59b5798add4e9dbfb5f5f2fc62756e30b5)
- [rename react-native-gradle-plugin to @react-native/gradle-plugin](https://github.com/facebook/react-native/commit/6f11b10a88235ad7de1a5777e5cdf7a582a231b7)
- [rename normalize-color to normalize-colors (umbrella 480)](https://github.com/facebook/react-native/commit/dc3355920d7f6f4ef887e0ff01153e23e660b5ea)
- [Bump remaining build-time Babel deps](https://github.com/facebook/react-native/commit/3e19c9764634049731b4bf96dc1c716064394d62)
- [Override default Talkback automatic content grouping and generate a custom contentDescription](https://github.com/facebook/react-native/commit/759056b49975c30cd561826e1499ebdf7aa8674d)
- [Remove ObjCTurboModule experiments with retaining JS callbacks in custom scopes](https://github.com/facebook/react-native/commit/229a1ded15772497fd632c299b336566d001e37d) - had to override TurboModule Files from this commit in ReactCommon [Created Issue](https://github.com/microsoft/react-native-windows/issues/11019)
- [Support colors in AnimatedInterpolation on Android](https://github.com/facebook/react-native/commit/e7dbfb2dbd98059ddd6e982e2e43c1e7df91a4cc) - This PR uses "matchAll" which is not supported in Chakra, temporary override the file until we hit [Replace matchAll with exec](https://github.com/facebook/react-native/commit/9b280ad1c5995f4d5bd5220dee778df3cd65db3f) which replaces the method [Created Issue](https://github.com/microsoft/react-native-windows/issues/11018)
- Removed override of RectangleEdges.h Resolves #10072

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/react-native-windows/pull/10999)

### CODE REVIEW

File: packages/@react-native-windows/codegen/package.json [hunk 1]
```diff
@@ -69,4 +69,4 @@
   "engines": {
     "node": ">= 14"
   }
-}
+}
```
Comment (dannyvv) [inline packages/@react-native-windows/codegen/package.json:72]: I had feedback on one of my PR's. that some unix tools are unhappy when there is no blank line at the end of a file...
I discovered that this was an auto-fixup we have configured in VsCode that runs on save...
I filed #11001  to fix this...
You'll have to either use notepad and add them back or disable 'format on save' temporary in VsCode settings file...

**Review by dannyvv — APPROVED** [review]

File: packages/@react-native-windows/codegen/package.json (hunk 1 above)
Comment (TatianaKapos) [reply packages/@react-native-windows/codegen/package.json]: Good to know! Thanks for catching that :)

**Comment by TatianaKapos** [comment]
Looks like this PR is hitting #11017, @ZihanChen-MSFT do you know what needs to be done to get the new code-gen files to work?

**Comment by acoates-ms** [comment]
I love the better type info coming from the codegen now.

**Review by acoates-ms — APPROVED** [review]

**Pull request merged**